unified-runtime/test/conformance/platform/urPlatformGet.cpp

```diff
@@ -12,43 +12,37 @@ struct urPlatformGetTest : ::testing::Test {
 };
 
 TEST_F(urPlatformGetTest, Success) {
-  uint32_t count;
-  ASSERT_SUCCESS(urPlatformGet(adapters.data(),
-                               static_cast<uint32_t>(adapters.size()), 0,
-                               nullptr, &count));
-  ASSERT_NE(count, 0);
-  std::vector<ur_platform_handle_t> platforms(count);
-  ASSERT_SUCCESS(urPlatformGet(adapters.data(),
-                               static_cast<uint32_t>(adapters.size()), count,
-                               platforms.data(), nullptr));
-  for (auto platform : platforms) {
-    ASSERT_NE(nullptr, platform);
+  for (auto adapter : adapters) {
```

Comment:
Other tests have one individual test per platform (e.g. one test for OpenCL, a different test for LevelZero, etc.). Might be worth doing that here as well so we can check if, for example, only OpenCL is failing this test.

I think this is something that's done by `uur::urAdapterTest` automagically.

Reply:
Great point. All fixed @RossBrunton .